src/Task1KtTest.kt

```diff
@@ -0,0 +1,16 @@
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Test
+import test.agileFreaks.Task1
+
+internal class Task1KtTest {
+
+    @Test
+    fun onEmptyArray() {
```

Comment:
You wrote to many tests for the first iteration. You should have only 1 failing test at once.